domenetjenester/behandling-revurdering/src/main/java/no/nav/foreldrepenger/behandling/revurdering/ytelse/fp/UttakGrunnlagTjeneste.java

```diff
@@ -123,6 +130,22 @@ public ForeldrepengerGrunnlag grunnlag(BehandlingReferanse ref) {
         return grunnlag;
     }
 
+    private boolean harMottattMorsArbeidDokument(Behandling behandling) {
```

Comment:
Vi må omarbeide denne litt for kommende søknader uten dokumentasjon.
Kalle den morsArbeidDokumentasjonIkkePåkrevdEllerMottatt ? 
Gitt lokal søknad sin 
    vedleggMorsArbeid = søknadRepo.hentSøknadFraGrunnlag(beh.id).map(getVedlegg).ellerTomListe
       filter(skjemanummer = MorArbeid).toList
Blir vel til at man kan registerVurdere dersom 
* vedleggMorsArbeid.isEmpty eller
* vedleggMorsArbeid.allMatch(lastetOpp) eller
* harMottattMorsArbeidDokument som i denne metoden

Reply:
Har lagt inn et forslag - du kan jo se

Reply:
Jeg synes det er bra løsning